### ISSUE TITLE
Should we keep LayerAccessTrait->linkedTo( )?

### ISSUE THREAD

**Issue by dreamingmind** [OrigamiStructures/StudioManagerScratch#133]
This was a very early tool in the suite; created before the whole system came into focus.

It duplicates functionality of the filter( ) process but was intended to provide an explicit, convenience method.

I find the signature a bit mysterious (though commenting could help) and it's not really come to mind as a valid option when I've been at the higher levels.

`linkedTo($foreign, $foreign_id, $linked = null)`

In a nutshell, it looks for '*foreign*_id' in the layer `$linked` and searches for the value `$foreign_id`.

This is the kind of multi-parameter call that layerAccessArgs was created to sort out.

Possibly some LAA methods for different associations could be written?

`$source->find($layer)->belongsTo($assoc, $assoc_id)->load()`
